scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/context/TaskContext.java

```diff
@@ -68,23 +71,21 @@
 
     private final String progressFilePath;
 
-    // for testing purpose only
-    public TaskContext(ExecutableContainer executableContainer, TaskLauncherInitializer initializer) throws
-      NodeException {
-        this(executableContainer, initializer, null, "", "", "", "", "", "",
-          "");
-    }
 
-    // for testing purpose only
-    public TaskContext(ExecutableContainer executableContainer,
-            TaskLauncherInitializer initializer, TaskResult[] previousTasksResults) throws NodeException {
-        this(executableContainer, initializer, previousTasksResults, "", "", "", "", "", "",
-          "");
+    public TaskContext(ExecutableContainer executableContainer, TaskLauncherInitializer initializer,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Method has 10 parameters, which is greater than 7 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00107)